src/igvfd/schemas/donor.json

```diff
@@ -67,6 +67,21 @@
             },
             "maxItems": 2
         },
+        "traits": {
+            "title": "Phenotypic traits",
+            "description": "A list of known phenotypic traits of the donor",
+            "type": "array",
+            "uniqueItems": true,
+            "default": [],
```

Comment:
I don't think we need an empty default array, I also think that minItem of the array should be 1.